Guided Investigation - MDATP Webshell Alerts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<p>Check iis_data is a populated DataFrame</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Enhanced error checking, thanks.</p>